webpack/assets/javascripts/react_app/components/charts/ChartHeader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+    constructor(props) {
+        super(props);
+        this.bind(
+            'onClick'
+        );
+    }
+
+    onClick() {
+        this.props.onClick();
+    }
+
+    render() {
+        // TODO: refactor to handle tooltip and placement here or in chart box
+        return (
+            <h4 className={this.props.cssClass}
```

Comment:
note, you should be using donut-title-big-pf css class instead, have a look at https://www.patternfly.org/patterns/donut-chart/ for more css reference